src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -111,7 +111,7 @@ export function PreSubmitSection(props) {
     formConfig?.customText?.finishAppLaterMessage ||
     FINISH_APP_LATER_DEFAULT_MESSAGE;
 
-  const saveFormLink = (
+  const saveFormLink = formConfig.disableSave ? null : (
```

Comment:
Even though we are opting into 0 "Save In Progress" form things, this "save for later" fallback is still hardcoded into the submit state errors.